src/ec.rs

```diff
@@ -558,11 +558,18 @@ where
       }
     }
     self.symbol(s, &cdf[..nsymbs]);
+    #[cfg(target_arch = "x86_64")]
+    {
     if cdf.len() == 5 && cfg!(target_feature = "sse2") {
       Self::update_cdf_4_sse2(cdf, s);
```

Comment:
Return here to remove the need for an else block.

Reply:
Sure